### PR TITLE
Clarifies format for a single pair of blackout dates

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-2245](https://openedx.atlassian.net/browse/DOC-2245)

Per support feedback, added a clearer description of what a single pair of blackout dates vs two pairs will look in the Discussion Blackout advanced setting.
### Reviewers

Possible roles follow. PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [x] Subject matter expert: @jaakana
- [x] Doc team review (copy edit): @srpearce 
### Testing
- [x] Ran en_us/make without warnings or errors
### Post-review
- [x] Squash commits

### CODE REVIEW

File: en_us/shared/building_and_running_chapters/running_course/discussions.rst [hunk 1]
```diff
@@ -748,33 +755,39 @@ Define When Discussions Are Closed
...
+    contains start and end dates in the following format.
 
-   When you enter the dates and times from the example above, the value field
-   looks like the following example.
+    ``[["2016-07-22T08:00", "2016-07-25T18:00"]]``
 
-   .. image:: ../../../shared/building_and_running_chapters/Images/Discussion_blackout_unformatted.png
-     :alt: Policy value of [["2014-07-22T08:00", "2014-07-25T18:00"],
-         ["2014-08-09T00:00", "2099-08-09T00:00"]].
+  * To add the dates that close the discussions permanently the field contains
```
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/running_course/discussions.rst]: Add comma after "permanently"

File: en_us/shared/building_and_running_chapters/running_course/discussions.rst [hunk 2]
```diff
@@ -748,33 +755,39 @@ Define When Discussions Are Closed
...
+    ``[["2016-07-22T08:00", "2016-07-25T18:00"], ["2016-08-09T00:00", "2099-08-09T00:00"]]``
 
-   Studio reformats your entry to add line feeds and indentation, like this:
+#. Select **Save Changes**.
 
-   .. image:: ../../../shared/building_and_running_chapters/Images/Discussion_blackout_formatted.png
-     :alt: Same policy value but with a line feed after each bracket and comma,
-         and an indent before each date.
+   Studio checks the syntax of your entry and reformats your entry to add line
+   feeds and indentation. A message notifies you of whether your changes were
```
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/running_course/discussions.rst]: "notifies you of" --> "lets you know

**Comment by srpearce** [comment]
Two nits; otherwise :+1:

**Comment by JAAkana** [comment]
Looks good. :palm_tree:

**Pull request merged**